src/applications/hca/sass/hca.scss

```diff
@@ -9,6 +9,19 @@
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
 @import "../../../platform/forms/sass/m-form-confirmation";
 
+.confirmation-page {
```

Comment:
I'm worried that adding this rule here will have an unintended impact the existing ConfirmationPage component.

Reply:
The className "confirmation-page" only exists in ConfirmationPageNew.jsx -- I verified this with a recursive folder-search under /src/applications/hca/.

If you're only concerned about styling impact, then this convo can be Resolved.  I'm leaving it unresolved **just in case** this has something to do with the current failed coverage test.  Not sure exactly what's causing that to fail.

Reply:
All tests have now passed, but leaving this convo to you to Resolve.

Reply:
Merge approved.